core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -96,6 +100,10 @@ class CoarseGrainedSchedulerBackendSuite extends SparkFunSuite with LocalSparkCo
       override def onTaskEnd(taskEnd: SparkListenerTaskEnd): Unit = {
         taskEnded.set(true)
       }
+
+      override def onExecutorAdded(executorAdded: SparkListenerExecutorAdded): Unit = {
```

Comment:
Please delete this method as not needed (inherited from SparkListener with the same body).

Reply:
Ah yes nice finding. I was about to add code there to test but changed my mind. It should be removed. Will remove.